src/components/ComboCount/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import './countShake.css';
 
 const ComboCount = ({ count }) => (
```

Comment:
Make sure to add the `comboCountClass` prop to the function signature

Reply:
added `comboCountClass` prop to the function signature